prow/gerrit/adapter/adapter_test.go

```diff
@@ -1111,8 +1208,25 @@ func TestProcessChange(t *testing.T) {
 				gc:            &gc,
 				tracker:       &fakeSync{val: fakeLastSync},
 			}
+			cloneURI, err := makeCloneURI(tc.instance, tc.change.Project)
+			if err != nil {
+				t.Errorf("error making CloneURI %v", err)
+			}
+
+			lg, cf, err := localgit.NewV2()
```

Comment:
since ProwYamlGetter is mocked, seems like this is unused?

Reply:
ProcessChange takes a clientFactory, and will only get to InRepoConfig if one is provided (if it is nil it will skip InRepoConfig). This is the best way to create one that is non-nil for the unit tests.

Reply:
would be nice to add a comment explaining it

Reply:
Done